app/views/schools/ects/change_mentor_wizard/check_answers.html.erb

```diff
@@ -0,0 +1,31 @@
+<% page_data(
+    title: "Check and confirm change",
+    backlink_href: @wizard.previous_step_path
+) %>
+
+<%= govuk_summary_list do |list| %>
+  <% list.with_row do |row| %>
+    <%= row.with_key { "Early career teacher" } %>
+    <%= row.with_value { @wizard.teacher_full_name } %>
+  <% end %>
+
+  <% list.with_row do |row| %>
+    <%= row.with_key { "Current mentor" } %>
+    <%= row.with_value { @wizard.current_step.current_mentor_name } %>
+  <% end %>
+
+  <% list.with_row do |row| %>
+    <%= row.with_key { "New mentor" } %>
```

Comment:
Do we need to render a lead provider row depending on if the user goes through that path?